packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts

```diff
@@ -100,9 +100,22 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
         }
       })
       .sort((a, b) => b.fiatAmount - a.fiatAmount)
-
+    const tokensWithBalance = tokens.filter((token) => token.balance.toString() !== '0')
```

Comment:
Why the `toString()`? Can it be `tokens.filter((token) => token.balance !== 0)`?

Reply:
token.balance is a BigNumber